addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14334,7 +14337,7 @@ msgstr ""
 #. Description of setting "Videos -> Playback -> Allow hardware acceleration (VDPAU)" with label #13425
 #: system/settings/settings.xml
 msgctxt "#36155"
-msgid "Enable VDPAU hardware decoding of video files, mainly used for NVIDIA graphics and in some circumstances AMD graphics."
+msgid "Enable VDPAU hardware decoding of video files, mainly used for Nvidia graphics and in some circumstances AMD graphics."
```

Comment:
Right, NVIDIA it is then.

> I have invested so many time in making German translation high quality with respect to consistency and "Everything" (tm). I have to admit that I'm really disappointed about that totally unneeded amount of work produced due to this kind of last second cleanups.

Is there anyway that a whole file can be reinserted and the corrections can be done manually in transifex?

Wow, this is really bad they dont leverage any versioning control whatsoever. very very bad. maybe its time to ditch transifex and use git 100%